routes/api.php

```diff
@@ -78,3 +78,10 @@
 Route::get('/groups/{group}/events', 'API\GroupController@getEventsForGroup');
 
 Route::get('/devices/{page}/{size}', [App\Http\Controllers\ApiController::class, 'getDevices']);
+
+// Notifications info.  We don't authenticate this, as API keys don't exist for all users.  There's no real privacy
+// issue with exposing the number of outstanding notifications.
+Route::get('/users/notifications/{id?}', 'API\UserController@notifications');
```

Comment:
Is id segment really optional?   We findOrFail in UserController::notifications so seems like we require an id.

Should the route perhaps be `/users/{id}/notifications`, as the id applies to the user resource, not the notification resource.

Reply:
Yes.  Changed.